src/Microsoft.Extensions.Configuration.Xml/XmlConfigurationProvider.cs

```diff
@@ -10,84 +10,15 @@
 namespace Microsoft.Extensions.Configuration.Xml
 {
     /// <summary>
-    /// An XML file based <see cref="ConfigurationProvider"/>.
+    /// An XML file based <see cref="FileConfigurationProvider"/>.
     /// </summary>
-    public class XmlConfigurationProvider : ConfigurationProvider
+    public class XmlConfigurationProvider : FileConfigurationProvider
     {
         private const string NameAttributeKey = "Name";
 
-        private readonly XmlDocumentDecryptor _xmlDocumentDecryptor;
+        internal XmlDocumentDecryptor Decryptor { get; set; } = XmlDocumentDecryptor.Instance;
```

Comment:
XmlDocumentDecryptor appears to be an internal class which looks only to be used for testing, hence the property is internal.  I guess we can make all of these things public if you prefer...